tests/integration/test_writes/test_writes.py

```diff
@@ -401,12 +402,12 @@ def test_python_writes_with_small_and_large_types_spark_reads(
     assert arrow_table_on_read.schema == pa.schema([
         pa.field("foo", pa.large_string()),
         pa.field("id", pa.int32()),
-        pa.field("name", pa.large_string()),
+        pa.field("name", pa.string()),
         pa.field(
             "address",
             pa.struct([
-                pa.field("street", pa.large_string()),
-                pa.field("city", pa.large_string()),
+                pa.field("street", pa.string()),
```

Comment:
totally outsider here but curious, was there a bug on pyarrow that made those `large_string` instead of `string`?

Reply:
@raulcd - It wasn't a bug, but actually an intentional change for the time being. If we update to PyArrow 17.0.0 we will be able to revert that change, and let the encoding in the parquet file dictate whether the table should be read as a large or small type for the Table API.